patches/installer/0005-Don-t-do-this-in-source-build-aspnetcore-targeting-p.patch

```diff
@@ -0,0 +1,35 @@
+From 5f34f12928157f5ed1c351dc217bf4bf5b62befe Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Mon, 14 Sep 2020 09:47:04 -0500
+Subject: [PATCH 5/5] Don't do this in source-build -
+ aspnetcore-targeting-pack.tar.gz not available.
```

Comment:
Is there an issue tracking this or a discussion about what this is/why source-build doesn't need it? Should loop in dotnet/installer folks if haven't been already.

Reply:
Logged https://github.com/dotnet/source-build/issues/1747 and I'll loop in installer.